lib/airbrake/rack/user.rb

```diff
@@ -50,7 +50,12 @@ def as_json
       private
 
       def try_to_get(key)
-        String(@user.__send__(key)) if @user.respond_to?(key)
+        String(@user.__send__(key)) if can_call_method?(key)
```

Comment:
I'm sorry for the confusion @valdis, however I'd like to make it clear that @xymbol doesn't maintain this repo and he's not our employee.

You can see that my badge says "Owner", and @xymbol's doesn't.

![don t call methods of user if parameters are required by valdis pull request 755 airbrake airbrake 2017-06-13 16-03-07](https://user-images.githubusercontent.com/1079123/27083619-d1f1df68-5051-11e7-8c3e-6ed8d623c122.png)